### PR TITLE
Wrap inputs in fieldsets in Space visibility settings

### PR DESCRIPTION
**Pull request by kerryarchibald**
https://element-io.atlassian.net/browse/PSE-118

Includes small visual changes to make Space settings more in line with Room settings:
- removal of inconsistent section separators (eg between preview and address heading)
- access radio styling in line with Room settings radio styling

Before:
Screenshot 2021-12-13 at 16 41 59
Screenshot 2021-12-13 at 16 41 51

After (updated):
Screenshot 2021-12-14 at 14 14 35

---
Here's what your changelog entry will look like:

## ✨ Features
 * Wrap inputs in fieldsets in Space visibility settings ([\#7350](https://github.com/matrix-org/matrix-react-sdk/pull/7350)). Contributed by @kerryarchibald.

Preview: https://61d6bf168b19e36cccaeadc4--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by t3chguy** [comment]
https://user-images.githubusercontent.com/3055605/145842554-0de32bdf-3b3c-4249-a5fb-8216122488f7.png

This is wrong, the `Show advanced` shouldn't be atop the radio interaction.

image

Colour/margins on `Recommended for public spaces` is also changed, as well as some other margins all around. Has this had design sign-off?

**Comment by kerryarchibald** [comment]
@gaelledel can you please take a look at the netlify to check the design? The changes are to space settings > visibility tab. Thanks!

File: src/components/views/spaces/SpaceSettingsVisibilityTab.tsx [hunk 1]
```diff
@@ -84,14 +89,9 @@ const SpaceSettingsVisibilityTab = ({ matrixClient: cli, space, closeSettingsFn
...
-            </>;
-        } else {
-            advancedSection = <>
-                <AccessibleButton onClick={toggleAdvancedSection} kind="link" className="mx_SettingsTab_showAdvanced">
-                    { _t("Show advanced") }
-                </AccessibleButton>
-            </>;
-        }
+            </div>
+            }
```
Comment (turt2live) [inline src/components/views/spaces/SpaceSettingsVisibilityTab.tsx:93]: The offset looks a bit awkward here:
```suggestion
            </div> }
```

**Review by turt2live — APPROVED** [review]
The code looks fine overall, just need design signoff :D

**Review by gaelledel — APPROVED** [review]
Yes except we need Quinary as a colour for separators 
So Dark/Quinary for Dark theme
Light/Quinary for Light theme

Reference here in Compound https://www.figma.com/file/X4XTH9iS2KGJ2wFKDqkyed/Compound?node-id=1893%3A42

**Comment by kerryarchibald** [comment]
Updated separators to quinary:
Screenshot 2022-01-06 at 10 48 57
Screenshot 2022-01-06 at 10 48 37

**Pull request merged**